applications/StructuralMechanicsApplication/custom_elements/cr_beam_element_3D2N.hpp

```diff
@@ -77,19 +82,54 @@ namespace Kratos
 
 		void Initialize() override;
 
+		/**
+         * @brief This function calculates the elastic part of the total stiffness matrix
+         */
 		bounded_matrix<double,msElementSize,msElementSize> CreateElementStiffnessMatrix_Material();

```

Comment:
Minor comment: while it is nice to have a function to return something, take care this is kind of risky from a performance point of view. You are relying on the optimizer to take out the temporary (however most of todays optimizers should do this ok)